src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>net452</TargetFrameworks>
```

Comment:
This is odd, as soon as I remove .Net 452 from this project file, VS builds, but it does not detect any unit test.